backend/estuary/estuary.go

```diff
@@ -436,12 +436,16 @@ func (f *Fs) List(ctx context.Context, dir string) (entries fs.DirEntries, err e
 			if err != nil {
 				return nil, err
 			}
-			o, err := f.newObjectWithInfo(ctx, path.Join(dir, item.Name), parseTimeString(pin.Meta["modTime"].(string)), &content{ID: item.ContentID, Cid: item.Cid, Size: item.Size})
+			modTime := pin.Meta["modTime"]
+			if modTime == nil {
+				// Only object created by rclone will have modTime
+				modTime = "0"
```

Comment:
can we change this to the current time or something? Let's default to something better than 0.

Reply:
removed